community/rfcs/GenAIExamples-24-05-16-001-Using_MicroService_to_implement_ChatQnA.md

```diff
@@ -0,0 +1,166 @@
+# Author
+liang1.lv@intel.com, feng.tian@intel.com, haihao.shen@intel.com, sihan.chen@intel.com, letong.han@intel.com 
+
+# Status
+Under Review
+
+# Objective
```

Comment:
I understand this RFC is to introduce OPEA microservice design and give an example on how to apply to RAG. It's better to mention this in objective as well.